test.yaml

```diff
@@ -1,17 +1,22 @@
-tests: 
+# Add the new parameters [here](tests/)
```

Comment:
Maybe we can add a comment here along with the link to the line where the parameters have to be added.